### ISSUE TITLE
Is the model trained using softmax appropriate for verification task?

### ISSUE THREAD

**Issue by heiwushi** [davidsandberg/facenet#787]
I downloaded the pretrained model and calculated the distance between the embeddings of two faces for comparing them. But the result is bad. Is it because the model is trained using softmax loss instead of tripletloss?   

Thank you!
